### DISCUSSION TITLE
form 嵌套 校验某个字段下所有的字段

### DISCUSSION THREAD

**Discussion by Lpbzzz** [ant-design/ant-design#44793] [Q&A] [answered]
Form 的结构是 
```
{
 a:{
   b1:[],
   b2:[],
   b3:[]
 },
 a2:{
  
 }
}
```

如果是 validateFields([ [ 'a','b1' ] ]) 是可以的校验指定的 b1 ，但是想一次校验所有的 a 下的字段呢？

**Comment by zombieJ** [comment, marked as answer]
<img width="543" alt="截屏2023-09-12 19 30 28" src="https://github.com/ant-design/ant-design/assets/5378891/73bbe2bd-869c-406c-8148-d30d7e656c40">
